src/locales/it.json

```diff
@@ -48,7 +48,7 @@
 "blw_service_link_label": "info@blw.admin.ch",
 "browse": "Browse",
 "cadastre": "Informazioni sui fondi",
-"cadastre_service_link_href": "http://www.cadastre.ch/internet/kataster/it/home.html",
+"cadastre_service_link_href": "https://www.cadastre.ch/en/home.html",
```

Comment:
@davidoesch it translation seems to be wrong in the google spreadsheet

Reply:
bugger , fixed. sorry